test/decl/var/didset_oldvalue_not_accessed.swift

```diff
@@ -0,0 +1,37 @@
+// RUN: %target-run-simple-swift | %FileCheck %s
```

Comment:
You need to also write a SILGen test and make sure you test global variables and members of generic types.

Reply:
Ok I'll add one now